common/logisticspipes/network/packets/SetCurrentPagePacket.java

```diff
@@ -31,7 +21,7 @@
 
 	@Getter
 	@Setter
-	private int page;
+	private int page, chapter, division;
```

Comment:
This will completely break when removing/modifying book pages. Use full page paths (as in, path to the page file), not indices

Reply:
I will do that in the Content getter but how would it be better to use that inside the code?

Reply:
Added to Glo Board